cmd/zdb/zdb.c

```diff
@@ -4418,9 +4418,9 @@ dump_l2arc_log_blocks(int fd, l2arc_dev_hdr_phys_t l2dhdr,
 		default:
 			abd = abd_alloc_for_io(asize, B_TRUE);
 			abd_copy_from_buf_off(abd, &this_lb, 0, asize);
-			zio_decompress_data(L2BLK_GET_COMPRESS(
+			VERIFY0(zio_decompress_data(L2BLK_GET_COMPRESS(
```

Comment:
I am not sure this should be VERIFY0(), not if () break alike to magic check below.

Reply:
It is in a switch statement. `break` means something else there. A goto would be the next best thing. I just revised this.